src/Tribe/Aggregator/Settings.php

```diff
@@ -270,12 +270,12 @@ public function default_settings_import( $origin ) {
 	public function get_url_import_range_options( $title = true ) {
 		$options = array(
 			DAY_IN_SECONDS       => array(
-				'title' => __( 'One day', 'the-events-calendar' ),
-				'range' => __( 'a day', 'the-events-calendar' ),
+				'title' => __( '24 hours', 'the-events-calendar' ),
```

Comment:
I really don't know why they exist, but I would say that this is up for discussion when @lucatume is back. He crafted this part of the code.